### ISSUE TITLE
dendron-cli: `dendron publish export` is broken; invalid regex

### ISSUE THREAD

**Issue by carmenbianca** [dendronhq/dendron#3367]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [ ] Pod.  Data import from / export to Dendron
- [x] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

It's impossible to run `dendron publish export` without errors.

## To Reproduce

1. Run `[npx] dendron publish export` on an existing Dendron repo.
2. See error.

## Expected behavior

Compiled output in `.next/out`.

## Screenshots

Output of command:

```
⬢ 📺 carmenbianca Dendron $ npx dendron publish export
🌱 checking if NextJS template is initialized
🌱 NextJS template is already initialized.
⠋ generating metadata for publishing...
⠼ {
  stack: 'DendronError: Multiple errors: \n' +
    ' - Failed to read docker.postgres.md in carmenbianca: Invalid regular expression: /\\bC++\\b/: Nothing to repeat\n' +
    ' - Failed to read huishouden.schoonmaken.md in carmenbianca: Invalid regular expression: /\\bC++\\b/: Nothing to repeat\n' +
[...]
  name: 'DendronError',
  status: 'unknown',
  severity: 'minor',
[...]
}
```

## Desktop (please complete the following information)

- OS: Fedora Silverblue 36
- Browser (if with published websites; include version): Irrelevant, but Firefox
- Versions (all that apply to bug):
  - Dendron Extension: Irrelevant
  - VS Code version: Irrelevant
  - Dendron CLI: 0.107.0

## Additional context

Temporary workaround:

Remove `'C++,'` line from `node_modules/title/lib/specials.js`.

## Dendron Log file

N/A

**Comment by hikchoi** [comment]
Apologies, we had a regression on `0.107.0`. This has been resolved with `0.107.2`. Could you update the CLI to `0.107.2` and let us know if that resolves the issue?

**Comment by carmenbianca** [comment]
Seems fixed to me. Thanks @hikchoi !
